superset/assets/src/visualizations/SatelliteToggle.css

```diff
@@ -0,0 +1,27 @@
+div.satellite-toggle {
+    font-size: 90%;
+    position: absolute;
+    background: #fff;
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.15);
+    margin: 24px;
+    /* padding: 12px 24px; */
+    outline: none;
+    overflow-y: scroll;
+    max-height: 200px;
+    top: 0px;
+    left: 0px;
+}
+
+button.satellite-toggle-button {
```

Comment:
Was wondering if it would be possible to use an icon for this in the same style as the icons on the right? It might be difficult to have a simple icon that indicates a satellite layer. Jonathan was the one who sorted the other icons, so can maybe help. If we keep the text I think we should remove the scroll bar and maybe change the text to be Sattelite when the map is showing and Map when the satellite is showing.

Reply:
I think those icons look good. I think it makes sense to move the icon to the right. To accomplish this I think it should be something similar to what is LayerSelector.css under the button.cbutton. 

Maybe something like this: 
    position: absolute;
    top: 110px;
    right: 8px;